content/docs/tutorials/azure/azure-ts-appservice-devops.md

```diff
@@ -2,9 +2,6 @@
 title: "A Todo App on Azure App Service with SQL Database and Application Insights and deploys it to Azure DevOps"
 ---
 
-<a href="https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/azure-ts-appservice-devops" target="_blank">
```

Comment:
I updated this example and raised a PR for it https://github.com/pulumi/examples/pull/380. I'll regenerate the tutorial once that is merged.

Reply:
The PR in the examples repo was merged.